### PR TITLE
Improve webui password reset text

### PR DESCRIPTION
**Pull request by ronnicek**
Updating text around webui password reset to make sure user understand he is going to loose everything.

### CODE REVIEW

File: troubleshooting.md [hunk 1]
```diff
@@ -139,6 +139,8 @@ and
 
 Do this only if you **do not have any funds** on your LND wallet!
 
+You are going to loose seed / settings / user data / application installed!
```
Comment (AaronDewes) [inline troubleshooting.md:142]: I think this is better, @louneskmt what do you think?
```suggestion
You are going to loose your seed, settings, data and applications!
```

Comment (louneskmt) [reply troubleshooting.md]: Also change the question to something like "Reset your Umbrel user data (if you lost your password)". We can also add a little sentence to tell the user that already has funds to keep its seed and backup file to be able to restore them later on.

**Review by AaronDewes — CHANGES_REQUESTED** [review]
See @louneskmt's comment (I didn't see it, sorry)

File: troubleshooting.md (hunk 1 above)
Comment (ronnicek) [reply troubleshooting.md]: Updated, feel free to check it :)

**Review by AaronDewes — APPROVED** [review]

**Review by louneskmt — APPROVED** [review]

**Pull request merged**